airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/registry_report.py

```diff
@@ -324,7 +324,6 @@ def connector_registry_report(context, all_destinations_dataframe, all_sources_d
 
     metadata = {
         "first_10_preview": MetadataValue.md(all_connectors_dataframe.head(10).to_markdown()),
-        "json": MetadataValue.json(json_string),
```

Comment:
what's going missing here?

Reply:
Exactly, we had the raw json there essentially for fun, but it seems thats a risk to the asset completing successfully. Now you just have to click the url below instead to see the same data

Reply:
word, that works!